FirebaseInstallations/Source/Tests/FIRInstallationsStoreTests.m

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import <OCMock/OCMock.h>
+
+#import <GoogleUtilities/GULUserDefaults.h>
+#import "FBLPromise+Testing.h"
+#import "FIRInstallationsErrorUtil.h"
+#import "FIRInstallationsItem.h"
+#import "FIRInstallationsStore.h"
+#import "FIRInstallationsStoredItem.h"
+#import "FIRSecureStorage.h"
+
+@interface FIRInstallationsStoreTests : XCTestCase
+@property(nonatomic) NSString *accessGroup;
+@property(nonatomic) FIRInstallationsStore *store;
+@property(nonatomic) id mockSecureStorage;
+@property(nonatomic) GULUserDefaults *userDefaults;
+@end
+
+@implementation FIRInstallationsStoreTests
+
+- (void)setUp {
+  self.accessGroup = @"accessGroup";
+  self.mockSecureStorage = OCMClassMock([FIRSecureStorage class]);
+  self.store = [[FIRInstallationsStore alloc] initWithSecureStorage:self.mockSecureStorage
+                                                        accessGroup:self.accessGroup];
+
+  // TODO: Replace real user defaults by an injected mock.
+  self.userDefaults =
```

Comment:
A better approach may be to have a suite named for each individual test name. That way there should be no collisions.

Reply:
TODO updated